templates/org/member/members.tmpl

```diff
@@ -1,49 +1,31 @@
 {{template "base/head" .}}
-<div role="main" aria-label="{{.Title}}" class="page-content organization members">
+<div role="main" aria-label="{{.Title}}" class="page-content organization">
```

Comment:
Why to delete this "members" class?

Reply:
Is there any impact? of regression

Reply:
Maybe they were depending on this class for customization. I wonder if there would be a better way to indicate the current page to CSS, like an data attribute that indicates the matching router or something. We could make better stability guarantees for those then these flimsy classes.

Reply:
> Is there any impact? of regression

Although there is currently no impact. But there is no way to tell under Vanilla JS whether it is a "Members" page.

Reply:
In JS, you could check the URL which I guess is more stable than these classes. e.g. check if url matches `\/[^/]+/[^/]+/members`.